Documentation/configuration/metrics.rst

```diff
@@ -29,6 +29,12 @@ Endpoint
 * ``endpoint_regenerating``: Number of endpoints currently regenerating
 * ``endpoint_regenerations``: Count of all endpoint regenerations that have completed, tagged by outcome
 
+Drops/Forwards (L3/L4)
+---------------------
+
+* ``drop_count_total``: Total drop requests, tagged by drop reason and ingress/egress direction
```

Comment:
Are these "drop requests", or "dropped packets"?

Reply:
Clearly packets

Reply:
I can change the wording to "packets"

Reply:
done